pkg/eval/sys/sys.go

```diff
@@ -0,0 +1,25 @@
+// Package sys exposes variables and functions that deal with the specific
```

Comment:
I am considering putting these variables related to interpreter internals in this module too:

https://github.com/elves/elvish/blob/7868b68ff34c7b0884b6f6e6520ee24b38f32b93/pkg/eval/eval.go#L148-L162

Which won't fit this description something along the description [Python's sys module](https://docs.python.org/3/library/sys.html) would be more accurate:

> This module provides access to some variables used or maintained by the interpreter and to functions that interact strongly with the interpreter.

Reply:
My preference is for anything that is explicitly tied to the OS or architecture, and which cannot be papered over by an OS agnostic API, be in the `os` namespace rather than in the builtin namespace.